packages/sdk-communication-layer/src/SocketService.ts

```diff
@@ -161,6 +161,10 @@ export class SocketService extends EventEmitter2 implements CommunicationLayer {
     return keyCheck(this);
   }
 
+  getKeyExchange() {
+    return this.state.keyExchange as KeyExchange;
+  }
```

Comment:
Add unit tests

Reply:
added the basic structure for covecod, will add full coverage later